ci/docker/debug-cuda-gh200-scalapack.yaml

```diff
@@ -37,6 +37,6 @@ spack:
         - '+fortran'
     pika:
       require:
-        - '@0.30.0'
+        - '@0.32.0'
```

Comment:
Is the plan to keep this explicitly at 0.32.0 or should this just use the latest version available?

Reply:
plan to keep 0.32.0 until we need to update

Reply:
Ok, sounds good. Should we still try to keep one (e.g. CPU) configuration on 0.30.0/1?